test/integration/cdi/example-driver/app/server.go

```diff
@@ -244,13 +244,25 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.AddCommand(controller)
 
+	// kubelet plugin
+	kubeletPluginFlagSets := cliflag.NamedFlagSets{}
+	fs = kubeletPluginFlagSets.FlagSet("kubelet plugin")
+
+	cdiAddress := fs.String("cdi-address", "/var/lib/kubelet/plugins/example-driver/cdi.sock", "Path to the CDI driver socket kubelet will use to issue CSI operations.")
```

Comment:
Would it be like KUBECONFIG? os.GetEnv?

Reply:
There is a command line parameter already (`--drivername`).

Reply:
Do you mean this one?
https://github.com/pohly/kubernetes/blob/dynamic-resource-allocation/test/integration/cdi/example-driver/app/server.go#L79

Umm,, I think I may still not know how to make things 'configurable'. I have been thinking that 'making them configurable' is doing like fs.String("something", "default value").

Is it not right? Then, I think your expression that I 'hard-coded' made me a bit confused. Because, anyway cdi-address is configurable by users?